scico/linop/radon_svmbir.py

```diff
@@ -230,15 +237,35 @@ def __init__(
         if prox_kwargs is None:
             prox_kwargs = dict()
```

Comment:
Egregious nitpick: I think ```prox_kwargs = {}``` would be a more standard way of writing this.

Reply:
Fixed.